tests/common/bridgecrew/test_wrapper.py

```diff
@@ -0,0 +1,29 @@
+
+def test_reduce_scan_reports_secrets(report):
+    from checkov.common.bridgecrew.wrapper import reduce_scan_reports
+    from checkov.common.typing import _ReducedScanReportCheck, _ReducedScanReport
+    from checkov.common.bridgecrew.check_type import CheckType
+
+    reduced_report: _ReducedScanReport = reduce_scan_reports([report])[CheckType.SECRETS]
+
+    checks: _ReducedScanReportCheck = reduced_report["checks"]
+    all_checks = checks["passed_checks"] + checks["failed_checks"] + checks["skipped_checks"]
+
+    assert all('validation_status' in check.keys() for check in all_checks)
+
+
+def test_reduce_scan_reports(report):
+    from checkov.common.bridgecrew.wrapper import reduce_scan_reports
+    from checkov.common.typing import _ReducedScanReportCheck, _ReducedScanReport
+    from checkov.common.bridgecrew.check_type import CheckType
+
+    report.check_type = CheckType.GITHUB_ACTIONS
```

Comment:
a small q :) why only test GHA?

Reply:
Doesn't matter, just chose a framework that is not secrets to verify that the validation_status doesn't appear in non-secrets reduced reports